specification/cognitiveservices/data-plane/Language/preview/2022-04-01-preview/textanalytics.json

```diff
@@ -2755,6 +3484,121 @@
+        },
+        "warnings": {
+          "type": "array",
+          "description": "Warnings encountered while processing document.",
+          "items": {
+            "$ref": "#/definitions/ConversationWarning"
+          }
+        },
+        "statistics": {
+          "$ref": "#/definitions/ConversationStatistics"
+        }
+      }
+    },
+    "ConversationError": {
+      "type": "object",
+      "required": [
+        "id",
+        "error"
+      ],
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "The id of the conversation."
+        },
+        "error": {
+          "type": "object",
+          "description": "Conversation Error.",
+          "$ref": "common.json#/definitions/Error"
+        }
+      }
+    },
+    "ConversationWarning": {
+      "type": "object",
+      "required": [
+        "code",
+        "message"
+      ],
+      "properties": {
+        "code": {
+          "type": "string",
+          "enum": [
+            "ConversationTruncated"
+          ],
+          "x-ms-enum": {
+            "name": "WarningCodeValue",
+            "modelAsString": true
+          },
+          "description": "Error code."
+        },
+        "message": {
+          "type": "string",
+          "description": "Warning message."
+        },
+        "targetRef": {
+          "type": "string",
+          "description": "A JSON pointer reference indicating the target object."
+        }
+      }
+    },
+    "ConversationStatistics": {
+      "type": "object",
+      "required": [
+        "transactionsCount"
+      ],
+      "properties": {
+        "transactionsCount": {
+          "type": "integer",
+          "format": "int32",
+          "description": "Number of transactions for the conversation."
+        }
+      },
+      "description": "If showStats=true was specified in the request this field will contain information about the conversation payload."
+    },
+    "ConversationRequestStatistics": {
+      "type": "object",
+      "required": [
+        "conversationsCount",
+        "validConversationsCount",
+        "erroneousConversationsCount",
+        "transactionsCount"
+      ],
+      "properties": {
+        "conversationsCount": {
+          "type": "integer",
+          "format": "int32",
+          "description": "Number of conversations submitted in the request."
+        },
+        "validConversationsCount": {
+          "type": "integer",
+          "format": "int32",
+          "description": "Number of conversations documents. This excludes empty, over-size limit or non-supported languages documents."
+        },
+        "erroneousConversationsCount": {
+          "type": "integer",
+          "format": "int32",
+          "description": "Number of invalid documents. This includes empty, over-size limit or non-supported languages documents."
+        },
+        "transactionsCount": {
+          "type": "integer",
+          "format": "int64",
```

Comment:
Why an `int64` when the rest are `int32`? Is this supposed to be like a total count and handle overflow of the summation of the other three? If so, perhaps `totalCount` would make more sense here? I'm just not clear on what you mean by "transaction" in this case. The description doesn't clarify either.

Reply:
It should be int32. Updated This is the billing unit for the operation.